### PR TITLE
fix for new line and international chars in header fields

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17117

## Description
Support new lines in headers (replace with white space). Support for encoding headers when needed for international text.

## Does it break backward compatibility?
  - No

## Must have
- [x] Tests
- [x] Documentation (release notes)
- [ ] Code Review

## Additional changes
Small refactoring to support unit tests.

### CODE REVIEW

File: Integrations/MailSenderNew/MailSenderNew.yml [hunk 1]
```diff
@@ -101,4 +101,5 @@ script:
     description: Send an email
   runonce: false
 tests:
-  - Mail Sender (New) Test
\ No newline at end of file
+  - Mail Sender (New) Test
+releaseNotes: "Support for Internationalized Headers. Fix for new lines in message subject."
```
Comment (yaakovi) [inline Integrations/MailSenderNew/MailSenderNew.yml]: release notes should be in a separate file now

Comment (yaakovi) [reply Integrations/MailSenderNew/MailSenderNew.yml]: (pull from master)

**Comment by glicht** [comment]
@yaakovi . The Header fixes the international stuff.

File: Integrations/MailSenderNew/MailSenderNew.yml (hunk 1 above)
Comment (glicht) [reply Integrations/MailSenderNew/MailSenderNew.yml]: Done

**Pull request merged**